src/cli/OutputGTFSCommand.ts

```diff
@@ -36,7 +36,8 @@ export class OutputGTFSCommand implements CLICommand {
     const transfersP = this.copy(this.repository.getTransfers(), "transfers.txt");
     const stopsP = this.copy(this.repository.getStops(), "stops.txt");
     const agencyP = this.copy(agencies, "agency.txt");
-    const fixedLinksP = this.copy(this.repository.getFixedLinks(), "links.txt");
+    // If excludeFixedLinks flag is set to true, we skip the get fixed links process here. 
+    const fixedLinksP = this.repository.excludeFixedLinks == true ? undefined : this.copy(this.repository.getFixedLinks(), "links.txt");
```

Comment:
Not sure if setting `fixedLinksP` to `undefined` is the best way to skip the the get fixed link process..

Reply:
We shouldn't just make this undefined, as this variable should actually be a Promise. In addition, I think we need to actually produce a `links.txt` file, as it's part of the GTFS spec.
There are two things we need to do here:
1. We need to add typing to these variables, so it would type error.
2. I'd move the ternary statement to be around `this.repository.getFixedLinks()`, so we either write out the fixed links or we write out an empty file.

Reply:
This is true, simply removing the `link.txt` will violate the GTFS spec. I have moved the ternary statement into `getFixedLinks()` method, if the flag is true, the method would return an empty list.

I also add typing for these variables in: 2c73484

Reply:
:+1: